dexcore/core.go

```diff
@@ -0,0 +1,92 @@
+package dexcore
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"time"
+
+	"decred.org/dcrdex/client/core"
+	"decred.org/dcrdex/dex"
+	libutils "github.com/crypto-power/cryptopower/libwallet/utils"
+)
+
+type Core interface {
+	Run(ctx context.Context)
+	Ready() <-chan struct{}
+	IsInitialized() bool
+	InitializeClient(pw, seed []byte) error
+	DiscoverAccount(dexAddr string, pass []byte, certI any) (*core.Exchange, bool, error)
+	BondsFeeBuffer(assetID uint32) uint64
+	PostBond(form *core.PostBondForm) (*core.PostBondResult, error)
+	NotificationFeed() <-chan core.Notification
+	Login(pw []byte) error
+}
```

Comment:
This should be defined by consumers, I think. Similar to how dcrdex/client/webserver defines a Core interface for methods that are needed by that package.